content/versions-comparison/1.7-news.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "title":"Teamwork Assistant",
+    "title":"Manage projects with the Teamwork Assistant",
```

Comment:
I want to avoid the active form with a verb, to avoid having complete sentences that are hard to read. It should begin by a noun.